tasks/connect.js

```diff
@@ -39,7 +39,15 @@ gulp.task('connect', function () {
         },
         cors(),
         history({
-          index: config.connect.historyFallbackIndex
+          index: config.connect.historyFallbackIndex,
+          rewrites: [
```

Comment:
We should have this array as the default value for a config like `historyRewrites` and then use it here like
```
rewrites: config.connect.historyRewrites
```

Reply:
@filipemartinsfe

Reply:
done @tiagomapmarques